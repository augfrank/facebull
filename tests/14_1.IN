M0   C0   C1  2102
M1   C0   C12  2160
M2   C1   C0  2020
M3   C1   C8  2992
M4   C1   C12  2523
M5   C2   C1  2298
M6   C2   C6  2910
M7   C3   C1  2004
M8   C3   C2  2484
M9   C3   C7  2054
M10   C3   C8  2096
M11   C4   C8  2870
M12   C4   C9  2895
M13   C5   C4  2558
M14   C5   C12  2166
M15   C5   C13  2499
M16   C6   C2  2416
M17   C6   C5  2338
M18   C6   C7  2404
M19   C7   C1  2191
M20   C7   C3  2486
M21   C7   C13  2822
M22   C8   C2  2827
M23   C9   C3  2314
M24   C10   C11  2467
M25   C11   C2  2689
M26   C11   C4  2296
M27   C11   C8  2082
M28   C12   C2  2988
M29   C12   C3  2336
M30   C12   C5  2790
M31   C13   C10  2717
