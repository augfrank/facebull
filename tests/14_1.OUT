34586
1 2 6 10 12 15 18 19 22 23 24 26 30 31
